opus/application/apps/ui/views.py

```diff
@@ -80,22 +81,23 @@ def get_context_data(self, **kwargs):
         return context
 
 @never_cache
-def api_last_blog_update(request):
-    """Return the date of the last blog update.
+def api_notifications(request):
+    """Return the HTML for any pending notifications and the date of the last blog update.
 
     This is a PRIVATE API.
 
-    Format: __lastblogupdate.json
+    Format: __notifications.json
 
     JSON return:
-        {'lastupdate': '2019-01-31'}
-      or if none available:
-        {'lastupdate': None}
+        {'lastupdate': '2019-01-31',                (or if none available 'None')
+         'notification': '<html code>',             (or if none available 'None')
+         'notification_mdate': '<file mod date>'   (store as cookie)
```

Comment:
"Store as cookie" is not relevant here - it's not part of the API. What the caller decides to do with this information is up to them.

Reply:
Also give an example of the mod date - what is the format?